docs/api.rst

```diff
@@ -2814,6 +2815,7 @@ Querying all balances
    :statuscode 400: Provided JSON is in some way malformed
    :statuscode 409: User is not logged in.
    :statuscode 500: Internal rotki error
+   :statuscode 502: Problem contacting a remote service
```

Comment:
as I wrote in discord this was not the task that raised. But the nft specific balances (and that already contains a docs entry)